website/plugins/docusaurus-plugin-open-graph-image/layout.js

```diff
@@ -0,0 +1,30 @@
+const { createSVGText } = require("./font");
+
+function createLayoutLayers(doc, layout, previewFont, textWidthLimit) {
+    const layers = layout.map((item) => {
+        if (!Object.prototype.hasOwnProperty.call(doc, item.name)) {
+            console.error(`Wrong template config? Doc property ${item.name} not found.`);
```

Comment:
[`nitpick:`](https://conventionalcomments.org/#labels) Мб так?
```suggestion
        if (!doc[item.name]) {
            console.error(`Wrong template config? Doc property ${item.name} not found.`);
```